src/transformers/models/auto/modeling_tf_auto.py

```diff
@@ -231,6 +231,7 @@
 
 TF_MODEL_FOR_VISION_2_SEQ_MAPPING_NAMES = OrderedDict(
     [
+        ("blip", "TFBlipForConditionalGeneration"),
```

Comment:
This needs to here, otherwise the script will compute the wrong `pipeline_model_mapping`.

cc @Rocketknight1: You forgot to add this one :-)

Reply:
I think it was missing for PyTorch as well, so I wasn't sure if I was supposed to!